.github/workflows/ci.yml

```diff
@@ -1,130 +1,109 @@
-name: hot-potato-ci
+---
+name: Hot Potato CI
 
 on:
-  push
-
-env:
-  isMaster: ${{ github.ref == 'refs/heads/master' }}
+  push:
```

Comment:
So I don't think we get free dependabot anymore, I added vulnerability scanning to this, but would it make sense to add in a cron job to run this on main weekly?